### ISSUE TITLE
Support for A-Frame > 0.7.0 

### ISSUE THREAD

**Issue by kfarr** [networked-aframe/networked-aframe#147]
Issue: NAF tracked controller example doesn't show Vive player updates when using A-frame latest version, but it does work as expected using 0.7.0 as currently in the example project.

Video recording: https://www.useloom.com/share/0c74b33672104627ab0b64c52dcc9a52

The video shows first the working example using 0.7.0 and then the "broken" example using A-Frame latest. Left side of screen is chrome "client" and right side is firefox "server" with htc vive.

**Comment by kfarr** [comment]
I tried a few more things:
* [Disabled LERPing](https://stackoverflow.com/questions/51997959/when-using-the-networked-a-frame-naf-component-how-do-i-disable-lerping/51997960#51997960), thought it might be related to a [lerping issue](https://github.com/haydenjameslee/aframe-lerp-component/issues/3) but still not seeing updates position and rotation from vive using a-frame 0.8.0 or above
* Funny enough using `wasd` controls on PC that is attached to Vive successfully moves player position which is synced correctly on other devices.

**Comment by wmurphyrd** [comment]
We use NAF with A-Frame v0.8.2, including sync of tracked controllers, without issue

**Comment by kfarr** [comment]
Hmm, does that example work for you too? Not sure what I'm doing wrong if it's just me :(

**Comment by kfarr** [comment]
Okay I figured it out, I think. I had not run `npm build` locally and `npm start` evidently does not provide a dynamically built version of build.js which meant that I was using the online version of `dist/networked-aframe.min.js` because of this line in the examples:
```
<script>window.NAF || document.write('<script src="https://unpkg.com/networked-aframe/dist/networked-aframe.min.js">\x3C/script>')</script>
```

Two fixes:
1. The repo needs the dist build updated. This is already covered in this PR #145 !! Please accept it! :)
https://github.com/networked-aframe/networked-aframe/pull/145/commits
2. I think the demo should be hard coded to use build.js and not give the option of the online version. That's probably a PR I am capable of if helpful...

**Comment by kfarr** [comment]
Resolved with #145  merged
